Remotion/Validation/UnitTests/RuleBuilders/AddingPropertyValidationRuleBuilderTest.cs

```diff
@@ -146,9 +147,15 @@ public void AddMetaValidationRule_ExpressionOverload_IsValidIsFalse ()
     }
 
     [Test]
-    [Ignore("TODO RM-5906")]
     public void SetCondition ()
     {
+      Func<Customer, bool> predicate = _ => true;
+
+      _addingPropertyValidationRuleCollectorMock
+          .Setup(mock => mock.SetCondition(predicate)).Verifiable();
```

Comment:
```suggestion
      _addingPropertyValidationRuleCollectorMock
          .Setup(mock => mock.SetCondition(predicate))
          .Verifiable();
```
Fluent Moq syntax gets alined 

Note: there's another way of writing this by adding the verify for the call at the end.
```
      _addingPropertyValidationRuleCollectorMock
          .Verify(mock => mock.SetCondition(predicate));
```

This way we don't use the setup and the general purpose Verify. It only works if no setup is required (i.e. no return value is needed). I tend to mix and mingle based on readability.

Reply:
okay, so we keep the existing syntax